fluent-release/yum/fluent-release.spec.in

```diff
@@ -0,0 +1,68 @@
+# -*- rpm -*-
+
+%define _amzn %{?amzn:%{amzn}}%{!?amzn:0}
+%define is_amazon_linux (%{_amzn} != 0)
+
+Name:		@PACKAGE@
+Version:	@VERSION@
+Release:	@RELEASE@%{?dist}
+Summary:	Fluent Package release files
+BuildArchitectures: noarch
+
+License:	ASL 2.0
+URL:		https://docs.fluentd.org/
+Source0:	@PACKAGE@-@VERSION@.tar.gz
+
+%if ! %{is_amazon_linux}
+Requires: epel-release
+Requires: dnf-command(config-manager)
+%else
+%if 0%{_amzn} == 2
+Requires: yum-utils
+%else
+Requires: dnf-command(config-manager)
+%endif
+%endif
+
+%description
+The stable distribution of Fluentd release repository files
+
+%prep
+%setup -q -n @PACKAGE@-%{version}
+
+%build
+
+%install
+
+mkdir -p %{buildroot}%{_sysconfdir}/pki/rpm-gpg/
+%{__install} -Dp -t %{buildroot}%{_sysconfdir}/pki/rpm-gpg/ -m0644 RPM-GPG-KEY-*
+
+mkdir -p %{buildroot}%{_sysconfdir}/yum.repos.d/
+
+if grep -q 'Amazon Linux' /etc/os-release 2>/dev/null; then
+  if grep -q 'VERSION="2"' /etc/os-release 2>/dev/null; then
```

Comment:
```suggestion
if grep -qs 'Amazon Linux' /etc/os-release; then
  if grep -qs 'VERSION="2"' /etc/os-release; then
```

Can we use `-s` (--no-messages) instead of `2>/dev/null`?